Mobl_ios_Callisto-master/MealTrackingPilot/AppDelegate.swift

```diff
@@ -21,6 +21,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     let primaryUserStorage = PilotPrimaryUserStorage()
 
+    
```

Comment:
Removed this extra 3 lines of spaces from AppDelegate.

Reply:
Done changes